### DISCUSSION TITLE
How to prevent window from stealing focus when the window's state changes?

### DISCUSSION THREAD

**Discussion by prateekamana** [qtile/qtile#3514] [Q&A] [answered]
https://user-images.githubusercontent.com/41776584/163756093-3c60cc3d-2e76-4076-a2c7-53c61c083aaa.mp4

Here we can see that the moment the browser window updates, it steals the focus from mpv player, then I manually take the focus back to mpv [mod+space], and new subtitles appear and the process repeats. How can I make it so that focus stealing is disabled? Any workaround would be really appreciated too.

**Comment by elParaguayo** [comment, marked as answer]
What value do you have for `focus_on_window_activation` in your config? Try setting it to `urgent` or `never` and see if that helps.

**Reply by prateekamana** [reply]
![image](https://user-images.githubusercontent.com/41776584/163805564-8760448e-02f4-4355-ab00-1ab37b2ed4ea.png)
I set it at the top of configuration like this, restarted qtile, but it didn't help any, is this how I was supposed to set this variable?

**Reply by prateekamana** [reply]
Ah, my bad it was being changed below it somewhere back to "smart", this fixed it, thank you very much good sir.
